lib/zendesk_apps_support/location.rb

```diff
@@ -35,7 +35,9 @@ def self.all
       Location.new(id: 7, orderable: false, name: 'background', product_code: Product::SUPPORT.code),
       Location.new(id: 8, orderable: true, name: 'chat_sidebar', product_code: Product::CHAT.code),
       Location.new(id: 9, orderable: false, name: 'modal', product_code: Product::SUPPORT.code),
-      Location.new(id: 10, orderable: false, name: 'ticket_editor', product_code: Product::SUPPORT.code)
+      Location.new(id: 10, orderable: false, name: 'ticket_editor', product_code: Product::SUPPORT.code),
+      Location.new(id: 11, orderable: false, name: 'top_bar', product_code: Product::STANDALONE_CHAT.code),
```

Comment:
I don't think this one needs to exist - the only app in Product::STANDALONE_CHAT will use system_top_bar right?